server/services/message-service.js

```diff
@@ -1,34 +1,59 @@
-const amqp = require('amqplib/callback_api')
+const rheapromise = require('rhea-promise')
 const config = require('../config')
+const calculationQueue = 'calculation'
+const scheduleQueue = 'schedule'
 
 module.exports = {
-  publishClaim: function (claim) {
-    try {
-      const messageQueue = config.messageQueue
-      amqp.connect(messageQueue, function (err, conn) {
-        if (err) {
-          console.log(err)
-        }
-        conn.createChannel(function (err, ch) {
-          if (err) {
-            console.log(err)
-          }
 
-          const data = JSON.stringify(claim)
+  publishClaim: async function (claim, messageQueueOptions) {
+    let configureMQ = function (options) {
+      return {
+        transport: options.transport,
+        port: options.port,
+        reconnect_limit: 10,
+        host: options.address,
+        hostname: options.address,
+        username: options.user,
+        password: options.pass
+      }
+    }
+    let sendClaim = async function (claim, connection, queueName) {

```

Comment:
I'm sure I left a comment for this, but seems to have disappeared, or didn't get saved.
The sendClaim and configureMQ functions should be moved outside as their own top level functions rather than be created as local functions each time publishClaim is called.